src/Sync/Singular.php

```diff
@@ -13,6 +13,18 @@ class Singular {
 
 	use CrudPropagation;
 
+	/**
+	 * In ElasticPress 5.0.0, the \ElasticPress\SyncManager::sync_queue attribute mutated:
+	 * - Before, it was an array indexed by post IDs with a TRUE value for those modified.
+	 * - After, those same entries got spread in parent arrays per site ID.
+	 *
+	 * For example, [ 123 => true ] became [ 1 => [ 123 => true ] ],
+	 * where 1 is the a block ID and 123 is a post ID for a post in the blog with ID equal 1.
+	 *
+	 * Nice for them to change some public data structure, right? Well, life!
+	 */
+	const SYNC_QUEUE_API_CHANGE_V1 = '5.0.0';
```

Comment:
@decodekult, this constant is already defined in `src/Plugin.php`.

Reply:
Yes, but I wanted to keep both values separate and only for the relevant class. One is about the change in the GUI sync mechanism, and the other one is about the signature change for the sync queue property. They happen on the same version, but they affect different areas, and I feel that we might need further dedicated "breakpoints" in the future for specific features.